app/views/members/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="w-full">
+
+  <div class="flex justify-between items-center">
+    <h1 class="font-bold text-4xl mb-5">Csapatunk</h1>
+    <% if policy(Member).new? %>
+      <%= link_to 'Tag hozzáadása', new_member_path, class: "btn btn-secondary" %>
+    <% end %>
+  </div>
+
+  <div class="flex flex-row flex-wrap">
```

Comment:
Maybe we should use a fixed grid
[https://tailwindcss.com/docs/grid-template-columns](https://tailwindcss.com/docs/grid-template-columns)

So that each member has the same size of card on the page